### PR TITLE
Address various window timeout bugs

### PR DESCRIPTION
**Pull request by openbots-su**

### CODE REVIEW

File: OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs [hunk 1]
```diff
@@ -69,6 +69,10 @@ public override void RunCommand(object sender)
 					Thread.Sleep(500);
 				}
             }
+			if (!User32Functions.GetActiveWindowTitle().Equals(windowName))
+			{
+				throw new Exception("Window Not Found");
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs:74]: could you add the window name variable to the exception message?

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs (hunk 1 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs]: Addressed in 3ad2290. Would you like me to add the window name to the timeout progress update messages as well?

Comment (openbots-ff) [reply OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs]: yeah, it's best if we can be specific as possible in the exception messages. Sometimes all anyone will have to go by is an execution log

File: OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs [hunk 2]
```diff
@@ -71,7 +71,7 @@ public override void RunCommand(object sender)
             }
 			if (!User32Functions.GetActiveWindowTitle().Equals(windowName))
 			{
-				throw new Exception("Window Not Found");
+				throw new Exception($"Window {windowName} Not Found");
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs:74]:
```suggestion
				throw new Exception($"Window '{windowName}' Not Found");
```

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs (hunk 1 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.UIAutomation/OpenBots.Commands.Window/ActivateWindowCommand.cs]: Addressed in d1055ef

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Pull request merged**